src/css/main.less

```diff
@@ -905,3 +909,43 @@ fieldset[disabled] .btn-info.active {
     padding: .8em 1em; /* this would be "10px 0" otherwise */
   }
 }
+
+.tab-pane.active.flex-container {
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  min-height: 100vh;
+
+  .wrapper {
+    display: flex;
+    flex: 1;
+
+    main {
+      flex: 1;
+      padding-left: (@base-spacing-unit * 2);
+    }
+
+    nav {
```

Comment:
Totally agree – what if instead of adding `nav.sidebar` we went for 

```
tab-pane.active.flex-container {
  // ...

  .wrapper {
    // ...
    
    > main {
    }
    
    >  nav {
    }
```
^ 
or is that too brittle?